patches.xml

```diff
@@ -2,4 +2,10 @@
 	<Executable Name="SLES_501.76;1" Title="Oni" Region="EU">
 		<Patch Address="0x001cef7c" Value="0x00000000 // bc0f $001cef7c" Description="Fix hang by skip branch if copro 0 condition false." />
 	</Executable>
+	<Executable Name="SCES_500.03;1" Title="Dead or Alive 2" Region="EU">
+		<Patch Address="0x002b4c44" Value="0x24060000" Description="Make the file reading RPC call synchronous. Boot hangs without this." />
+	</Executable>
+	<Executable Name="SLPS_250.02;1" Title="Dead or Alive 2" Region="JP">

```

Comment:
The SLPS one is a guess as I don't have that version. Strelok confirmed it worked though ... just not sure if the region is correct or if it even matters.